build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+import { fail } from "assert";
```

Comment:
Doesn't look like this is used?

Reply:
Accidentally left `import` statement after trying out some assert statements. 😄